packages/eslint-plugin-udemy/README.md

```diff
@@ -1,7 +1,7 @@
 Udemy ESLint Plugin
 ===================
 
-Udemy specific linting rules for ESLint.
+This package has Udemy specific ESLint rules.
```

Comment:
Udemy-specific

Reply:
I don't think this deserves to be dash-separated.

Reply:
@cansin

When you have two words used together as an adjective, you add a hyphen. It's an "advanced" rule that a lot of people don't necessarily get right, so it's okay if you ignore it.

Example: Vim is the best modal-based editor in existence.